File: Linguagem de Programação I/Extras/Beecrowd/2770.py
import sys

for line in sys.stdin:
    X, Y, M = map(int, line.split())
    for _ in range(M):
        Xi, Yi = map(int, sys.stdin.readline().split())
        if (Xi <= X and Yi <= Y) or (Xi <= Y and Yi <= X):
            print("Sim")
        else:
            print("Nao")

File: Linguagem de Programação I/Extras/Beecrowd_Extra/2770.cpp
#include <iostream>

int main() {
    int X, Y, M, Xi, Yi;

    while (std::cin >> X >> Y >> M) {
        for (int i = 0; i < M; ++i) {
            std::cin >> Xi >> Yi;
            if ((Xi <= X && Yi <= Y) || (Xi <= Y && Yi <= X)) {
                std::cout << "Sim\n";
            } else {
                std::cout << "Nao\n";
            }
        }
    }

    return 0;
}